src/commands/git-metadata/upload.ts

```diff
@@ -117,8 +125,13 @@ export class UploadCommand extends Command {
         metricsLogger.logger.increment('gitdb.success', 1)
         this.logger.info(`${this.dryRun ? '[DRYRUN] ' : ''}Successfully synced git DB in ${elapsed} seconds.`)
       } catch (err) {
-        console.log('error writing to git db')
-        this.logger.warn(`Could not write to GitDB: ${err}`)
+        if (!this.isTargetingGov()) {
+          console.log('error writing to git db')
+          this.logger.warn(`Could not write to GitDB: ${err}`)
+        } else {
+          // Skip the warning for Gov DC since git sync is not available there yet.
+          this.logger.warn(`not writing to GitDB: not available for gov`)
```

Comment:
nit: You are mixing sentences starting with uppercases and lowercases
```suggestion
          this.logger.warn(`Not writing to GitDB: not available for gov`)
```